tests/scripts/webhook-config.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
```

Comment:
can this just be a symlink if the RBAC is moved to common.yaml as suggested?

Reply:
Moved to tests/scripts as wrapper script has been added